### PR TITLE
First few MiniHack environments

### PR DESCRIPTION
**Pull request by samvelyan**
This branch is branched from `vit-level-gen` so includes Vit's additions as well.

### CODE REVIEW

**Comment by yobibyte** [comment]
Thanks! I'll review this after ICML deadline (tomorrow).

File: nle/env/minihack.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    This environment is an empty room, and the goal of the agent is to reach
+    the staircase, which provides a sparse reward.  A small penalty
+    is subtracted for the number of steps to reach the goal. This environment
+    is useful, with small rooms, to validate that your RL algorithm works
+    correctly, and with large rooms to experiment with sparse rewards and
+    exploration.
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs["max_episode_steps"] = kwargs.pop("max_episode_steps", 50)
```
Comment (yobibyte) [inline nle/env/minihack.py:84]: Did you omit the room size by intent? 

[MiniGrid](https://github.com/maximecb/gym-minigrid/blob/67926ede9d857895501691ec4559f432ce8912f8/gym_minigrid/envs/empty.py#L20) changes the number of steps based on the room size.

File: nle/env/tasks.py [hunk 2]
```diff
@@ -496,6 +498,32 @@ def __init__(self, *args, **kwargs):
 
         self.n_distractors = kwargs.pop("inventory_distractors", 0)
         self.randomise_n_distractors = kwargs.pop("randomise_num_distractors", True)
+
+        # TODO this is automated now, but still hardcoded, stay tuned
+        level_description = """# NetHack 3.6	oracle.des	
+#
+
+LEVEL: \"oracle\"
```
Comment (yobibyte) [inline nle/env/tasks.py:506]: This change looks weird since it's exactly the same code I wrote. How did this got merged into your branch?

**Review by yobibyte — APPROVED** [review]
Looks good to me, but I am not sure why some commits are shown as added lines since they are exactly the same as added by me in another branch.

File: nle/env/tasks.py (hunk 2 above)
Comment (samvelyan) [reply nle/env/tasks.py:506]: See my comment below.

**Comment by yobibyte** [comment]
Ah, my bad. I somehow missed that.

**Pull request merged**